docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -160,6 +160,8 @@ Noteworthy module changes
 * The ``digital_ocean`` module has been deprecated in favor of modules that do not require external dependencies.
   This allows for more flexibility and better module support.
 
+* The ``docker_swarm_service`` module will no longer set a default for the user option.
```

Comment:
```suggestion
* The ``docker_swarm_service`` module no longer sets a default for the ``user`` option. Before, the default was ``root``.
```
Mentioning the old default makes it easier for people to adjust their playbooks/roles to the old behavior :)